devops/artifactory/creating_backup_sources_repo.rst

```diff
@@ -0,0 +1,49 @@
+.. _devops_artifactory_backup_sources_repo:
```

Comment:
Lets move it to devops/backup_sources/repositories/artifactory

It could be "servers", "repos", or other similar name instead of "repositories"